Doc/library/sqlite3.rst

```diff
@@ -937,13 +946,25 @@ Connection objects
 
    .. method:: setlimit(category, limit, /)
 
-      Set a connection runtime limit. *category* is the limit category to be
-      set. *limit* is the new limit. If the new limit is a negative number, the
-      limit is unchanged.
-
+      Set a connection runtime limit.
       Attempts to increase a limit above its hard upper bound are silently
-      truncated to the hard upper bound. Regardless of whether or not the limit
-      was changed, the prior value of the limit is returned.
+      truncated to the hard upper bound.
+      Regardless of whether or not the limit was changed,
+      the prior value of the limit is returned.
+
+      :param category:
+          The `SQLite limit category`_ to be set.
+      :type category: int
```

Comment:
I haven't used this syntax yet, but by looking at the examples at https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#python-signatures, it seems that the following should work:
```suggestion
      :param int category: The `SQLite limit category`_ to be set.
```

In order to improve the vertical space economy, I would try whenever possible to:
* combine `param` and `type` as suggested above
* remove empty lines between the various `:*:`
* possibly add the description next to the `:*:` (even though for longer description I'm not sure how it would work)

Reply:
Oh, TIL! That's nice. We should definitely use those for the reasons you provide.

Reply:
FYI, I'm updating this PR to also apply this markdown shortcut to the existing `:param`'s that need it.

Reply:
> possibly add the description next to the `:*:` (even though for longer description I'm not sure how it would work)

Actually, I'm not so sure about this. IMO, it looks a little bit more messy when reading the raw file. I'm leaning on keeping the descriptions always on the line below.

Reply:
I've reverted the concatenations; this also reduces the diff.